src/vue/components/word-list/word-language-panel.vue

```diff
@@ -118,9 +126,34 @@ export default {
         } else {
           console.warn(`The current filter method - ${this.selectedFilterBy} - is not defined, that's why empty result is returned!`)
         }
-      }
+      }      
       return []
     },
+    wordExactForms () {
+      let exactForms = []
+      this.wordlist.values.map(wordItem => {
```

Comment:
I think would be slightly better to use `reduce` for that. The following code will probably be simpler to read, on my opinion:
```
let exactForms = this.wordlist.values.reduce((acc, wordItem) => {
    let exactForm = wordItem.targetWord.toLowerCase()
    if (exactForms.includes(exactForm)) {
      acc.push(exactForm)
   }
}, [])
```

Reply:
Fixed